### ISSUE TITLE
deprecation warning about using `distutils.util`

### ISSUE THREAD

**Issue by jugmac00** [flyingcircusio/batou#159]
```
❯ ./batou deploy staging
Removing expired path: .batou/42f59059 ...
Creating venv ...
/home/jugmac00/Projects/bliss_deployment/./batou:78: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  import distutils.util
...
```

**Comment by ctheune** [comment]
This is Python 3.10, right?

**Comment by jugmac00** [comment]
Yes

**Comment by elikoga** [comment]
Removed in appenv in https://github.com/flyingcircusio/appenv/pull/52, will be included in batou with #458
